src/libs/kata-types/src/mount.rs

```diff
@@ -64,6 +64,9 @@ pub const KATA_VIRTUAL_VOLUME_LAYER_NYDUS_FS: &str = "layer_nydus_fs";
 /// Download and extra container image inside guest vm.
 pub const KATA_VIRTUAL_VOLUME_IMAGE_GUEST_PULL: &str = "image_guest_pull";
 
+/// force_guest_pull enforces guest pull without remote snapshotter but the prerequsitive setting of `shared_fs = none` should be ensured.
```

Comment:
> but the prerequsitive setting of `shared_fs = none` should be ensured.

As far as I understand, this is not really related to `force_guest_pull` but advisable for all CoCo setups - right?